osbrain/proxy.py

```diff
@@ -100,6 +100,30 @@ def _ready_or_timeout(self, time0, timeout):
             raise
         return True
 
+    def wait_for_running(self, timeout=0):
+        """
+        Wait until the agent is running.
+
+        Parameters
+        ----------
+        timeout : float
+            If different than 0, raise an error if the agent is not running
+            after that number of seconds.
```

Comment:
What if we say: "if greater or equal than 0" (or "if not negative")?

0 seconds timeout could make sense as well. In example: just see if the agent is running once and otherwise immediately raise. -1 seconds timeout makes no sense and is many times used for non-negative parameters that could be ignored.

Reply:
I didn't think 0 seconds made any sense, but OK, I'll change it to "less than zero"